db/Controllers/api.controllers.js

```diff
@@ -1,4 +1,5 @@
+const { fetchApiInstructions } = require("../Models/api.models")
 
-exports.getApiInstructions = () => {
-    fet
+exports.getApiInstructions = (req,res,next) => {
+    res.status(200).send(fetchApiInstructions())
```

Comment:
at this point - you should either require in the endpoints.json or fs.readFile - sending it to a model isn't really ideal.